src/components/ProjectIndex/ProjectIndex.js

```diff
@@ -37,11 +37,7 @@ const ProjectIndex = (props) => {
           ButtonIcon={PlusIcon}
         />
       </ProjectIndexHeader>
-      <div className='main-container'>
-        {projectList.map((project, i) => (
-          <ProjectListItem project={project} user={user} key={i} />
-        ))}
-      </div>
+      <ProjectListTable projects={projectList} user={user}/>
     </>
```

Comment:
Also, do we want the header to show when the projects are still loading? Might also reduce flashing content later down the line if we do? And (maybe this is out of scope sorry) but can we put the Loading text in the translations file?

Reply:
Yuuuup, great idea! Done in [509f2a3](https://github.com/RaspberryPiFoundation/editor-ui/pull/317/commits/509f2a3e73fb2b2221dea8e4aa876037c099aa8c) (Added translation as well :-))